charts/juicefs-csi-driver/values.yaml

```diff
@@ -50,6 +50,11 @@ mountMode: mountpod
 # The name of the JuiceFS CSI driver
 driverName: "csi.juicefs.com"
 
+# CSIDriver resource configuration
+csidriver:
+  # Disable Helm hooks on CSIDriver resource. Set to true to prevent CSIDriver deletion/recreation during Helm upgrades. This is useful for ArgoCD and GitOps workflows where hook behavior can cause the CSIDriver resource to randomly go missing during sync operations.
+  disableHooks: false
```

Comment:
Can we disable this hook by default? cc @zwwhdls

Reply:
LGTM